src/main/scala/chisel3/simulator/PeekPokeAPI.scala

```diff
@@ -13,18 +14,30 @@ object PeekPokeAPI {
     def step(cycles: Int = 1): Unit = {
       val context = currentContext()
       context.willEvaluate()
-      if (cycles == 0) {
-        context.controller.run(0)
-      } else {
-        val simulationPort = context.simulationPorts(clock)
-        simulationPort.tick(
-          timestepsPerPhase = 1,
-          maxCycles = cycles,
-          inPhaseValue = 0,
-          outOfPhaseValue = 1,
-          sentinel = None
-        )
-      }
+      val simulationPort = context.simulationPorts(clock)
+      simulationPort.tick(
```

Comment:
This has slightly different semantics (the initial state of the clock port is set) which I think are more correct (encountered during testing).